include/sdf/Model.hh

```diff
@@ -348,8 +348,14 @@ namespace sdf
 
     /// \brief Create and return an SDF element filled with data from this
     /// model.
+    /// \param[in] _useUri When true, the model's URI is used to create an
+    /// SDF `<include>` rather than a `<model>`. The model's URI must be
+    /// first set using the `Model::SetUri` function. If the model's URI is
+    /// empty, then a `<model>` element will be generated. The default is true
+    /// so that URI values are used when ToElement is called from a
+    /// World object.
```

Comment:
Can you also add that calling `Model::SetUri` is required even if the model is originally loaded from an `<include>`?

Reply:
Done in 1eb2d0e